src/pybind/mgr/dashboard/frontend/src/app/shared/api/prometheus.service.ts

```diff
@@ -200,36 +206,71 @@ export class PrometheusService {
     return this.http.get<any>(`${this.baseURL}/data`, { params });
   }
 
-  getMultiClusterQueriesData(selectedTime: any, queries: any, queriesResults: any) {
+  getMultiClusterQueriesData(
+    selectedTime: any,
+    poolsChartSelectedTime: any,
+    clusterUtilizationChartSelectedTime: any,
```

Comment:
You need to different time values you need to mount them as different objects and it will work.